darts/timeseries.py

```diff
@@ -4076,7 +4076,7 @@ def _set_freq_in_xa(xa_: xr.DataArray):
 
             # restore a RangeIndex if needed:
             time_idx = xa_.get_index(self._time_dim)
-            if isinstance(time_idx, pd.Int64Index) and not isinstance(
+            if isinstance(time_idx, pd.Index) and not isinstance(
```

Comment:
pd.DatetimeIndex() is also an instance of pd.Index. For series with a DatetimeIndex, we would return a new time series with a Range Index instead of a DatetimeIndex. So I believe we need to check the following:
```suggestion
            if time_idx.is_integer() and not isinstance(
```